### PR TITLE
Add vote_unmute and display on it chat (Fixes #1376)

### LINKED ISSUE #1376: Vote ban system is incomplete
There is no `vote_unmute` nor a command to list current vote bans.
Also, I think vote bans should be announced in chat. Shows others that fun-voting has consequences and adds moderator transperincy.

### PR DESCRIPTION
**Pull request by Ryozuki**
Fixes #1376

### CODE REVIEW

File: src/game/server/gamecontext.h [hunk 1]
```diff
@@ -387,7 +388,8 @@ class CGameContext : public IGameServer
 	CVoteMute m_aVoteMutes[MAX_VOTE_BANS];
 	int m_NumVoteMutes;
 	void Mute(const NETADDR *pAddr, int Secs, const char *pDisplayName);
-	void VoteMute(const NETADDR *pAddr, int Secs, const char *pDisplayName, int AuthedID);
+	bool VoteMute(const NETADDR *pAddr, int Secs, const char *pDisplayName, int AuthedID);
+	bool VoteUnMute(const NETADDR *pAddr, const char *pDisplayName, int AuthedID);
```
Comment (def-) [inline src/game/server/gamecontext.h]: Unmute instead of UnMute

**Comment by def-** [comment]
Doesn't fix #1376 fully, two more parts remain.

**Comment by Ryozuki** [comment]
> Doesn't fix #1376 fully, two more parts remain.

one is listing mutes and the other?

**Comment by def-** [comment]
Thanks, hope to see more people becoming more active again :)

File: src/game/server/ddracecommands.cpp [hunk 2]
```diff
@@ -438,8 +460,72 @@ void CGameContext::ConVoteMute(IConsole::IResult *pResult, void *pUserData)
...
+	{
+		net_addr_str(&pSelf->m_aVoteMutes[i].m_Addr, aIpBuf, sizeof(aIpBuf), false);
+		str_format(
+				aBuf,
+				sizeof aBuf,
+				"%d: \"%s\", %d seconds left",
+				i,
+				aIpBuf,
+				(pSelf->m_aVoteMutes[i].m_Expire - pSelf->Server()->Tick())
+				/ pSelf->Server()->TickSpeed());
```
Comment (def-) [inline src/game/server/ddracecommands.cpp]: This formating is a bit unusual for TW style.

Comment (Ryozuki) [reply src/game/server/ddracecommands.cpp]: I copied it from mutes, but if u want i can fix both

**Comment by def-** [comment]
> one is listing mutes and the other?

Sorry, you got them all.

**Pull request merged**